### ISSUE TITLE
[1.18.2] Game crashes when breaking animation starts with shaders enabled

### ISSUE THREAD

**Issue by poisax** [Asek3/Oculus#110]
### Minecraft Version

1.18.2

### Iris Version

oculus-mc1.18.2-1.2.5.jar

### Sodium Version

rubidium-0.5.2a.jar

### Operating System

Windows 11

### What is your GPU?

Nvidia GeForce RTX 2060

### Java Version

Java 17

### Reproduction Steps

1. Start breaking a block in survival
2. As soon as the animation is supposed to play
3. Game Crashes

### Crash Report file and latest.log

```shell
https://gist.github.com/poisax/30b133c9a179dea0a9a202e58ff91835
```

### Additional context

There is no crash report, only the logs. Also the game process keeps running in the background

**Comment by InteJason** [comment]
https://paste.ee/p/pl87r
same issue

Edit: Found that this issue happens when you update forge beyond 40.1.23

**Comment by SDUBZ** [comment]
i noticed myself this seems to only happen with nvidia cards

**Comment by SDUBZ** [comment]
okays i got the crash from watching a friend and my cards a amd rx 590 so its just not nvidia
like i thought
[debug.log](https://github.com/Asek3/Oculus/files/9029953/debug.log)
[latest.log](https://github.com/Asek3/Oculus/files/9029954/latest.log)
[crash-2022-07-02_01.20.55-client.txt](https://github.com/Asek3/Oculus/files/9029957/crash-2022-07-02_01.20.55-client.txt)

@Asek3

**Comment by archonsd** [comment]
This happens for me too with any version of Forge above .47.  I can reliably cause it by upgrading only Forge to any of .51, .52, and .54.  Downgrading back to 1.18.2-40.1.47 allows it to not recur.

Oculus: oculus-mc1.18.2-1.2.5.jar
Rubidium:  rubidium-0.5.2a.jar
Forge (happens, tested):  forge-1.18.2-40.1.51, .52, and .54
Forge (stable):  forge-1.18.2-40.1.47

Update:  Can confirm that disabling shaders (SEUS PTGI HRR 3 in my case) but leaving the same revisions of Oculus and Rubidium in place allows block breaking to occur normally without a crash.

**Comment by SDUBZ** [comment]
if we had the new tweaks to iris in this mod or one of its addons
we could disable the block breaking particle and it should be useable again

**Comment by newbie12121** [comment]
Likewise getting this issue.
https://paste.ee/p/uqZ5p crash report

**Comment by Gavin456b** [comment]
Also having this issue

**Comment by GeggsLegs** [comment]
I am also getting it with complementary shaders
on forge 40.1.60 and on forge 40.1.67

**Comment by Gbergz** [comment]
Can confirm.

**Comment by krumen** [comment]
Same here, this need to be fix'd plz.
Nvidia 3080, tested with 8 shaders, every time, same error:

Forge 40.1.67

The game crashed whilst tesselating block model
Error: java.lang.IllegalArgumentException: Vertex element out of bounds: 6

**Comment by Gavin456b** [comment]
Yep, still having this issue in forge 40.1.67 whenever shaders are enabled. I can't downgrade my version of Oculus either because then it bugs out my shaders.

**Comment by ChloeDawn** [comment]
This crash occurs when the experimental Forge light pipeline is enabled. As a temporary workaround, the light pipeline can be disabled in `~/config/forge-client.toml` by setting `experimentalForgeLightPipelineEnabled` to `false`, but may cause rendering bugs with custom models.

**Comment by Gavin456b** [comment]
Thank you! I will try this out and see if it fixes the problem

**Comment by Gavin456b** [comment]
> This crash occurs when the experimental Forge light pipeline is enabled. As a temporary workaround, the light pipeline can be disabled in `~/config/forge-client.toml` by setting `experimentalForgeLightPipelineEnabled` to `false`, but may cause rendering bugs with custom models.

Okay so I disable it but it turns itself back on by default whenever I start the game.

**Comment by ChloeDawn** [comment]
One of your mods might be forcing it back on, I'm not sure

**Comment by Gavin456b** [comment]
Well it does fix the problem I just have to force it off whenever I start the game
